pro-planner-client/src/components/UserSideBar.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { useState } from 'react';
+import { Card, Col, ListGroup, Row } from 'react-bootstrap';
+
+const UserSideBar = props => {
+	const userList = useSelector(state => state.user.userList);
+	const loggedInUser = useSelector(state => state.user.selectedUser);
+	const selectedUser = props.selectedUser;
+	const setSelectedUser = props.setSelectedUser;
+	const isEditing = props.isEditing;
+
+	return (
+		<div className="d-flex flex-column justify-content-center align-items-center mt-4">
+			<Card className="mt-2" style={{ width: '350px' }}>
+				<Card.Header as="h4" className="text-center">
+					Responders
+				</Card.Header>
+				<Card.Body>
+					<ListGroup
+						variant="flush"
+						style={{ maxHeight: '300px', overflow: 'scroll' }}
```

Comment:
I would say having a vertical scroll is good for overflow, but maybe we shouldn't have a horizontal scroll, and rather just ellipses names that are too long for our component?